postcss.config.js

```diff
@@ -1,6 +1,19 @@
+const path = require('path');
+const { execSync } = require("child_process");
+
+const themeRoot = execSync(`bundle show bullet_train-themes-${process.env.THEME} 2> /dev/null`).toString().trim()
```

Comment:
I'm noticing that if I create a new theme after running the `init_local_css` rake command, [bin/theme:22](https://github.com/bullet-train-co/bullet_train/blob/c803363a618acbd109fa322511a0544868a97941/bin/theme#L22) will try to look for a gem that doesn't exist because I only ejected the files and didn't create an entirely new gem. Should we create a condition over in `bin/theme` that looks for the paths in the local app if the gem doesn't exist?

```
> rake bullet_train:themes:light:init_local_css
> rake bullet_train:themes:light:eject[foo]
```

```
foo-css        | node:internal/errors:862
foo-css        |   const err = new Error(message);
foo-css        |               ^
foo-css        | 
foo-css        | Error: Command failed: bundle show bullet_train-themes-foo 2> /dev/null
foo-css        |     at checkExecSyncError (node:child_process:885:11)
foo-css        |     at execSync (node:child_process:957:15)
foo-css        |     at Object.<anonymous> (/home/gazayas/work/bt/bullet_train/postcss.config.js:4:19)
foo-css        |     at Module._compile (node:internal/modules/cjs/loader:1255:14)
foo-css        |     at Module._extensions..js (node:internal/modules/cjs/loader:1309:10)
foo-css        |     at Module.load (node:internal/modules/cjs/loader:1113:32)
foo-css        |     at Module._load (node:internal/modules/cjs/loader:960:12)
foo-css        |     at Module.require (node:internal/modules/cjs/loader:1137:19)
foo-css        |     at require (node:internal/modules/helpers:121:18) {
foo-css        |   status: 7,
foo-css        |   signal: null,
foo-css        |   output: [ null, Buffer(0) [Uint8Array] [], Buffer(0) [Uint8Array] [] ],
foo-css        |   pid: 18383,
foo-css        |   stdout: Buffer(0) [Uint8Array] [],
foo-css        |   stderr: Buffer(0) [Uint8Array] []
foo-css        | }
foo-css        | 
foo-css        | Node.js v20.2.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
foo-css        | Exited
```

Reply:
Potentially related to https://github.com/bullet-train-co/bullet_train-core/issues/727, will have to look over that one too.